api-overview/about-overview.md

```diff
@@ -2,43 +2,43 @@
 
 ### When Do I Need an Overview?
 
-Every document set should include an overview. This document tells potential users or buyers of the product the things they might want to know before adoption.
+Your API documentation should always include an overview document. This document tells potential users or buyers what they might want to know before deciding to use your API.
 
 Many users install or plug into your service immediately because they are already familiar with it or someone else has decided that the team will use this solution. All of those people will go to your Quickstart, or will just get to work.
 
-There are other groups, however, who need to assess your product in order to decide whether they want to use it. They are either the managers responsible for a business decision, or they are engineers who don't know enough about your product to decide whether it can accomplish what they want it to do. People who fall in either of these groups might decide not adopt your product unless you provide an overview that answers their questions.  
+There are other groups, however, who need to assess your product in order to decide whether they want to use it. For example, managers responsible for a business decision or engineers responsible for a technical assessment of your product. People in these groups might decide to not adopt your product unless you provide an overview that answers their questions.
 
 ## Content of Your Overview
 
 ### Who is This Overview For?
 
-* Are you writing only for developers? For managers?
-* Are you writing only for people who have a certain problem to solve?
-* Is it intended for a particular industry.
+-   Are you writing only for developers? For managers?
+-   Are you writing only for people who have a certain problem to solve?
+-   Is it intended for a particular industry.
```

Comment:
I don't see the point in switching from bullet point formats. Unless we have set a convention across all docs, (which I don't think we have done), I suggest keeping the same as before, so you don't introduce extra review time for people.
Ditto for other lists in this file.

Reply:
This was my mistake, my formatter automatically converted it and I didn't notice, I'll convert it back. I think we should add some kind of formatter config so that keeping it consistent over many contributions becomes easy